### PR TITLE
Post workout data

### PR DESCRIPTION
**Pull request by BrandonMathis**
## What Changed
* Created endpoint for android app to post data to

## Why
* So that we can keep up with type of activity

### CODE REVIEW

File: app/controllers/activities_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class ActivitiesController < ApplicationController
+  def create
+    Activity.transaction do
+      params[:activities].each do |activity|
+        @last_activity = Activity.new(activity_params(activity))
```
Comment (iandonovan) [inline app/controllers/activities_controller.rb]: Is there a reason for the instance variable here that gets saved on the next line? Why not just do `Activity.create!(activity_params(activity))`?

File: app/controllers/activities_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        @last_activity.save!
+      end
+    end
+
+  rescue ActiveRecord::RecordInvalid
+    render json: @last_activity.errors.to_hash
```
Comment (iandonovan) [inline app/controllers/activities_controller.rb]: Ah yes lol okay

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  resources :activities, except: [:new, :edit]
   root to: 'air_quality_observations#index'
   namespace :api do
     resources :air_quality_observations, except: [:new, :edit]
+    resources :activities, only: [:post, :index]
```
Comment (iandonovan) [inline config/routes.rb]: This is neat syntax; I've always done `only: [:create, :index]` but if it recognizes `:post` then :+1:

**Comment by iandonovan** [comment]
:shipit:

File: config/routes.rb (hunk 3 above)
Comment (BrandonMathis) [inline config/routes.rb]: hmmm, I think I like `:create` more tho

**Pull request merged**